Runar/Runar/Controllers/Generator flow/RunicDrawsCollection/RunicDrawsCollectionVC.swift

```diff
@@ -85,6 +85,10 @@ class RunicDrawsCollectionVC: UICollectionViewController {
     override func viewWillAppear(_ animated: Bool) {
         super.viewWillAppear(animated)
         self.tabBarController?.tabBar.isHidden = false
+        if UserDefaults.standard.bool(forKey: "hasViewedOnboardingScreen") {
+            return
+        }
+        self.navigationController?.pushViewController(OnboardingScreenVC(), animated: true)
```

Comment:
Why do you use OnboardingScreenVC()?
It is more readable to create object and transfer it by the ref.